app/controllers/import_users_controller.rb

```diff
@@ -56,6 +56,13 @@ def valid_input_file?(imported_users, file) # rubocop:disable all
       redirect_to(:back) && return
     end
 
+    # make sure that there's spaces in the header line
+    if imported_users.first.keys.join('').match(/\s/)
```

Comment:
Oooh I really like that idea, as long as we always ensure that our headers have no whitespace (which they shouldn't, since they're effectively object attributes). Let's go with that in #1299.